packages/fxa-admin-panel/src/components/Nav/index.tsx

```diff
@@ -66,6 +66,23 @@ export const Nav = () => (
             Admin Logs
           </NavLink>
         </li>
+        <li>
```

Comment:
I took the liberty of adding this new section. Perhaps it was outside the scope of this ticket, but the section seems useful for subsequent tasks, and to some degree it validates the user / group is being set on the client side.